packages/platform-android/src/commands/runAndroid/adb.ts

```diff
@@ -41,6 +41,57 @@ function getDevices(adbPath: string): Array<string> {
   }
 }
 
+function getAllAvailableDevices(adbPath: string) {
```

Comment:
Three questions:
1/ Is this more `getDevicesAndEmulators`, in other words, an array of devices & emulators that are not booted?
2/ Why not extend `getDevices` already? Seems a bit odd to introduce another one. If you want to have a separate one, why not to call `adb.getDevices()` on line 59?
3/ Why not use `parseDevicesResult` here, but instead custom logic? Is there something we can do to tweak that function to be useful here as well?